charts/nvidia-installer/templates/ds-k8s-nvidia-deviceplugin.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "nvidia-device-plugin.fullname" . }}
+  namespace: {{ .Values.nvidiaDevicePlugin.namespace }}
+  labels:
+    app: {{ template "nvidia-device-plugin.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  template:
+    metadata:
+      labels:
+        name: {{ template "nvidia-device-plugin.name" . }}
+        k8s-app: {{ template "nvidia-device-plugin.name" . }}
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      priorityClassName: system-node-critical
+      volumes:
+      - name: device-plugin
+        hostPath:
+          path: {{ required "nvidia-device-plugin.hostDevicePluginPath" .Values.nvidiaDevicePlugin.hostDevicePluginPath }}
+      - name: dev
+        hostPath:
+          path: /dev
+      containers:
+      - image: "k8s.gcr.io/nvidia-gpu-device-plugin@sha256:0842734032018be107fa2490c98156992911e3e1f2a21e059ff0105b07dd8e9e"
```

Comment:
I would make the images also configurable via the `values.yaml` and set this one as default there.

Reply:
done